physionet-django/console/utility.py

```diff
@@ -13,8 +13,9 @@
 from django.contrib.sites.models import Site
 from django.conf import settings
 from django.urls import reverse
-
-from project.validators import validate_doi
```

Comment:
tests are failing because`from project.validators import validate_doi` was removed. it needs to be added back.

Reply:
@tompollard I have added the import (from project.validators import validate_doi) and I'm still getting this error